src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -36,33 +36,36 @@ class NotebookCard extends Component {
     const { namespace, name, updated, listView, wsName, onRename, onCopy, onDelete, canCompute, canWrite } = this.props
 
     const notebookMenu = canWrite && h(PopupTrigger, {
-      position: 'bottom',
+      position: 'right',
       closeOnClick: true,
       content: h(Fragment, [
         h(MenuButton, {
           onClick: () => onRename()
-        }, ['Rename']),
+        }, [icon('renameIcon', { size: 15, style: { margin: '0 .25rem 0 0' } }), 'Rename']),
         h(MenuButton, {
           onClick: () => onCopy()
-        }, ['Duplicate']),
+        }, [icon('copy', { size: 15, style: { margin: '0 .25rem 0 0' } }), 'Clone']),
         h(MenuButton, {
           onClick: () => onDelete()
-        }, ['Delete'])
+        }, [icon('trash', { size: 15, style: { margin: '0 .25rem 0 0' } }), 'Delete'])
```

Comment:
nit: There's just enough duplication here that it might be nice to create a little local helper for this.